packages/create-sitecore-jss/src/templates/nextjs/src/lib/next-config/plugins/cors-header.js

```diff
@@ -10,6 +10,7 @@ const corsHeaderPlugin = (nextConfig = {}) => {
   return Object.assign({}, nextConfig, {
     async headers() {
       return [
+        ...await nextConfig?.headers(),
```

Comment:
Shouldn't need `?` here right? Given the default on line 6.
```suggestion
        ...await nextConfig.headers(),
```

Reply:
If there isn't headers param in next.config.js there will be error about `nextConfig.headers is not a function`

Reply:
But this would still be a problem with this code since the optional chaining only applies to the preceding prop (`nextConfig`), right? To handle undefined `headers`, think we'd want something like this:
https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-styleguide/src/lib/next-config/plugins/disconnected.js#L16

Reply:
@ambrauer I added some changes because if there isn't custom headers in next.config.js there will show error. I added conditional for fix this situation